single_steps_submits/fastp.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#Submit to the cluster, give it a unique name
+#$ -S /bin/bash
+
+#$ -cwd
+#$ -V
+#$ -l h_vmem=1.9G,h_rt=20:00:00,tmem=1.9G
+#$ -pe smp 2
+
+# join stdout and stderr output
+#$ -j y
+#$ -R y
+
+if [ "$1" != "" ]; then
+    RUN_NAME=$1
+else
+    RUN_NAME=$""
+fi
+
+FOLDER=$(date +"%Y%m%d%H%M")
+WRITEFOLDER=../submissions/$FOLDER
+mkdir -p $WRITEFOLDER
+
+snakemake -s single_steps/fastp.smk \
+--jobscript cluster_qsub.sh \
+--cluster-config config/cluster.yaml \
+--use-conda \
+--cluster-sync "qsub -l tmem={cluster.tmem},h_vmem={cluster.h_vmem},h_rt={cluster.h_rt} -o $FOLDER {cluster.submission_string}" \
+-j 40 \
+--nolock \
+--rerun-incomplete \
+--latency-wait 100
```

Comment:
Don't necessarily need this, if you name the configs correctly can use the base `submit.sh`. would prefer to do that to keep the repo clean